SecretManagement.LastPass.Extension/SecretManagement.LastPass.Extension.psm1

```diff
@@ -52,16 +52,53 @@ $SpecialKeys = @('Language', 'NoteType', 'Notes')
 
 #endregion
 
+# Internal wrapper around the actual call. Reserved for Invoke-lpass. Use Invoke-lpass instead.
+function Invoke-lpassInternal {
+    [CmdletBinding()]
+    Param(
+        [Parameter(ValueFromPipeline)]
+        [object]$InputObject,
+        [Switch]$UseNative,
+        [String]$lpassPath,
+        [string[]]$Arguments
+    )
+
+    if ($UseNative) {
+        if ($InputObject) { return $InputObject | & $lpassPath @Arguments }
+        return & $lpassPath @Arguments
+    }
+    # WSL
+    if ($InputObject) { return $InputObject | & wsl $lpassPath @Arguments }
+    return & wsl $lpassPath @Arguments
+}
+
+
+<#
+.SYNOPSIS
+Manage calls to lastpass.
+
+.DESCRIPTION
+Manage calls to lastpass. show & login command are special and requires us to not redirect streams
+to get the prompt. Everything else have its error stream redirect to success (2>&1)
+
+.PARAMETER Arguments
+Arguments to pass to lpass CLI
+
+.PARAMETER InputObject
+Used by some of the non-interactive commands such as lpass add / lpass edit
+
+.PARAMETER VaultParams
+This should never be populated from the extension. It is only for the main module, which have
+no awareness of the vault parameters and need to have them fed to it. If this parameter is provided,
+it overrides $AdditionalParameters
+#>
 function Invoke-lpass {
     [CmdletBinding()]
     param (
         [Parameter()]
-        [string[]]
-        $Arguments,
-
+        [string[]]$Arguments,
         [Parameter(ValueFromPipeline)]
-        [object]
-        $InputObject,
+        [object]$InputObject,
         #Only for root module functions
         [hashtable]$VaultParams
     )
```

Comment:
Half of the parameter were on the same line `[String]$Param` and the other was dual line. 
So I made that consistent. I personally prefer having the type and param on the same line (easier to read / less scrolling) but if you prefer the other way around, I can change it so it's the other way around (but consistent)